pkg/storage/backend-storage/backend-storage.go

```diff
@@ -0,0 +1,101 @@
+package backendstorage
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/utils/pointer"
+
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	PVCPrefix = "persistent-state-for-"
+	PVCSize   = "10Mi"
+)
+
+func PersistentPVCForVMI(vmi *corev1.VirtualMachineInstance) string {
```

Comment:
nit: would it be better to name the function, e.g. `BackendStoragePVCforVMI`? Because PVC already stands for *persistent* volume claim. So we have 'persistent' twice here.

Reply:
This is already the `backendstorage` package, so just `PVCForVMI()` should do.